src/coreComponents/physicsSolvers/inducedSeismicity/QuasiDynamicEQ.cpp

```diff
@@ -0,0 +1,282 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2016-2024 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2024 Total, S.A
+ * Copyright (c) 2018-2024 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2023-2024 Chevron
+ * Copyright (c) 2019-     GEOS/GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file QuasiDynamicEQ.cpp
+ */
+
+#include "QuasiDynamicEQ.hpp"
+
+#include "dataRepository/InputFlags.hpp"
+#include "mesh/DomainPartition.hpp"
+#include "kernels/RateAndStateKernels.hpp"
+#include "rateAndStateFields.hpp"
+#include "physicsSolvers/contact/ContactFields.hpp"
+#include "fieldSpecification/FieldSpecificationManager.hpp"
+
+namespace geos
+{
+
+using namespace dataRepository;
+using namespace fields;
+using namespace constitutive;
+
+QuasiDynamicEQ::QuasiDynamicEQ( const string & name,
+                                Group * const parent ):
+  SolverBase( name, parent ),
+  m_stressSolver( nullptr ),
+  m_stressSolverName( "SpringSlider" ),
+  m_maxNewtonIterations( 2 ),
```

Comment:
2 is enough?

Reply:
not always but in any case I need to use teh one in NonLinearSolverParameters instead of this one